### ISSUE TITLE
Manually alter the bit rate

### ISSUE THREAD

**Issue by BoweryCZ** [mangui/flashls#456]
We are using commercial version of FlowPlayer incl. HLS plugin. HLS plugin is multistream. Is it possible to manually alter the bit rate using Java Script or via plugin „Menu“?

**Comment by phloxic** [comment]
No. At least not the 'elegant' way, i.e. with smooth switching with the means of flashls. Only with the bit rate selection plugin http://flash.flowplayer.org/plugins/streaming/brselect.html - but that would mean re-starting the stream because the core plugins are aware of flashls; not what you're looking for, I guess.

Manual hls bit rate selection will be implemented for Flowplayer HTML5 in the hlsjs plugin - https://github.com/flowplayer/flowplayer-hlsjs - we just need time, but right now don't have it ;-)

**Comment by BoweryCZ** [comment]
Thank you very much for your quick answer. We are really looking for the new function. We have one more question. We have 2 HLS video clips in a playlist in Flowplayer. The first video item plays OK (play/pause/stop/resume). The second video item has a bug. When we pause it and then resume, video part slowly fades away, only audio part continues to play without problem. The problem is not in the video clips, we tryied to switch them, the same result.

Please take a look at the sample:
http://embed.livebox.cz/ta3_test/vod-resize.html

Best regards and thank you for your time

**Comment by phloxic** [comment]
@BoweryCZ - note that, as I said, the manual switching will _not_ be implemented for Flowplayer Flash (at least not by us), but in the hlsjs plugin for Flowplayer HTML5: https://flowplayer.org/docs/plugins.html#hlsjs

As to your setup: With flashls you either have to use a hardware accelerated setup with `wmode: "direct"` in the Flash configuration and `accelerated: true` in the clip configuration - see both demos on the flashls plugin page: http://flash.flowplayer.org/plugins/streaming/flashls.html - or, if you must use opaque or transparent wmode (always has a negative performance impact), you have to set `hls_usehardwarevideodecoder: false` in the flashls plugin configuration.
Background: flashls defaults to hw acceleration, Flowplayer core does not.
